test-network/scripts/utils.sh

```diff
@@ -24,16 +25,18 @@ function printHelp() {
     println "    -h - Print this message"
     println
     println " Possible Mode and flag combinations"
-    println "   \033[0;32mup\033[0m -ca -r -d -s -verbose"
-    println "   \033[0;32mup createChannel\033[0m -ca -c -r -d -s -verbose"
+    println "   \033[0;32mup\033[0m -ca -bft -r -d -s -verbose"
+    println "   \033[0;32mup createChannel\033[0m -ca -bft -c -r -d -s -verbose"
```

Comment:
This is in contradiction with the note added "that currently this sample does not yet support the use of consensus type BFT and CA together", isn't it?